src/js/scrollArea.jsx

```diff
@@ -148,6 +163,11 @@ export default class ScrollArea extends React.Component{
         this.setState(newState);
     }
 
+    convertLinesToPixels(lines) {
+        var pixels = lineHeight(findDOMNode(this.refs.content));
```

Comment:
Is that necessary to compute line height on every handleWheel event? Maybe you could move it to `componentDidMount` and compute line height only once